src/filetree.ts

```diff
@@ -419,10 +420,23 @@ export function constructFileTreeWidget(
   restorer: ILayoutRestorer,
   manager: IDocumentManager,
   router: IRouter,
+  labShell: ILabShell,
 ) {
   const widget = new FileTreeWidget(app, basepath, id || "jupyterlab-filetree");
   restorer.add(widget, widget.id);
-  app.shell.add(widget, side);
+  app.shell.add(widget, side, { rank: 0 });
```

Comment:
We often create several filetrees, so I don't think we want `rank: 0` here. Maybe something like `rank: number_of_current_filetrees` so they pop up after the last one.